onnxscript/tests/models/pnp.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+
+import onnx
+from onnx import TensorProto
+from onnx.helper import make_tensor
+
+from onnxscript import script, graph
+# from onnxscript.onnx_opset import opset18ext as op
+from onnxscript.onnx_types import FLOAT, INT64
+
+from typing import (
+    Any,
+    Callable,
+    Mapping,
+)
+
+from onnxscript.onnx_opset._impl.opset18 import Opset18
+from onnxscript import tensor as onnxscript_tensor
+from onnxscript.onnx_types import (
+    BFLOAT16,
+    BOOL,
+    COMPLEX64,
+    COMPLEX128,
+    DOUBLE,
+    FLOAT,
+    FLOAT16,
+    INT8,
+    INT16,
+    INT32,
+    INT64,
+    STRING,
+    UINT8,
+    UINT16,
+    UINT32,
+    UINT64,
+)
+from onnxscript.values import Op, Opset
+from typing import Callable, Optional, Sequence, Tuple, Union
+
+import numpy as np
+import onnxruntime as rt
+import onnx
+
+class Opset18Ext(Opset18):

```

Comment:
I need a custom operator so that it can be programmatically edited to replace the op with the core model graph. I should not has used onnx domain though.